src/object/srv_obj.c

```diff
@@ -3073,7 +3083,7 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 	if (ioc.ioc_map_ver < max_ver)
 		ioc.ioc_map_ver = max_ver;
 
-	obj_rw_reply(rpc, rc, epoch.oe_value, &ioc);
+	obj_rw_reply(rpc, rc, epoch.oe_value, false, &ioc);
```

Comment:
Why not early reply input buffer for update on leader target? Is there any overhead for input buffer early reply? If not, why not early reply for all object RPCs to the engine without distinguish whether it is from client or another server?

Reply:
@gnailzenh 's part as well, will let him answer

Reply:
From what he said yesterday in the google call, I suspect there are some latent bugs that prevent it.   Ideally, we should be able to do this on every rpc if I understand correctly but it causes some issues if we do that.

Reply:
> Why not early reply input buffer for update on leader target? Is there any overhead for input buffer early reply? If not, why not early reply for all object RPCs to the engine without distinguish whether it is from client or another server?

Because we ran into some issues when we tried to apply this to all RPCs, now we are trying to do this step by step and only apply this change to the most significant option of RPCs as the first step.